common/buildcraft/lib/BCMessageHandler.java

```diff
@@ -53,6 +55,8 @@ public static void fmlPreInit() {
 
     public static void fmlPostInit() {
         netWrapper = NetworkRegistry.INSTANCE.newSimpleChannel("bc-lib");
+        netWrapper.registerMessage(MessageZonePlannerMapChunkRequest.Handler.class, MessageZonePlannerMapChunkRequest.class, 123, Side.SERVER); // TODO: move it form here
+        netWrapper.registerMessage(MessageZonePlannerMapChunkResponse.Handler.class, MessageZonePlannerMapChunkResponse.class, 124, Side.CLIENT);
```

Comment:
This should be in the BCRobotics class, in ````preInit````. It should *probably* look like this:
````java
BCMessageHandler.addMessageType(MessageZonePlannerMapChunkRequest.class, MessageZonePlannerMapChunkRequest.Handler.INSTANCE, Side.SERVER);
BCMessageHandler.addMessageType(MessageZonePlannerMapChunkResponse.class, MessageZonePlannerMapChunkResponse.Handler.INSTANCE, Side.CLIENT);
````

This does require you to change the handlers to enum types, with a single object "INSTANCE".